packages/ui/src/composites/wui-list-account/styles.ts

```diff
@@ -0,0 +1,43 @@
+import { css } from 'lit'
+
+export default css`
+  button {
+    padding: 6.5px var(--wui-spacing-l) 6.5px var(--wui-spacing-xs);
```

Comment:
I know this is more of design question but just to double check if we are okay to use some value that is not in our spacing system. (I mean `6.5px`) We can skip this if it's a designers choice bc we generally want to stick to the design variables

Reply:
copied it from similar components, this value (6.5px) is used on quite few places but can update ofc if required

Reply:
@ganchoradkov Then I guess we could keep it I'd like to not block your time for UI details. We should review those in general